lib/readGltfGeometry.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
I was also wondering if I should drop the `gltf` and go for `primitive` so: `convertPrimitiveToGeometry` and `convertGeometryToPrimitive`

or even just `primitiveToGeometry` and `geometryToPrimitive`